common-util/src/main/java/org/sunbird/common/responsecode/ResponseMessage.java

```diff
@@ -411,6 +411,9 @@
     String INVALID_PAGE_SECTION = "Page section associated with the page is invalid.";
     String ERROR_RATE_LIMIT_EXCEEDED =
         "Your per {0} rate limit has exceeded. You can retry after some time.";
+    String ERROR_DIAL_CODE_NOT_RESERVED = "Dial code: {0} not reserved for textBook id: {1}.";
+    String ERROR_INVALID_TOPIC =
+        "Topic <{0}> not found in the framework. Please check and correct.";
```

Comment:
If we know framework id or framework name  - we should include in message

Reply:
this is the message suggested on ticket .